docs/platform/getting-started/overview.mdx

```diff
@@ -18,13 +18,8 @@ Your setup might be a little different depending on which solution you chose.
 Getting started with Falcon Platform can be broken down into a few simple steps.
 
 1. [Create a Falcon Platform account](prerequisites)
```

Comment:
Do you think the step name should contain the page name somehow (in this case prerequisites). I feel like it could be confusing that it does not.

Reply:
done